src/components/toast/toast.component.js

```diff
@@ -9,7 +9,7 @@ import {
   ToastStyle, ToastTypeStyle, ToastContentStyle, ToastWrapper
```

Comment:
no cypress tests at all for golden border

Reply:
added in: bf7d8ab